content/en/actions/actions_catalog/_index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Actions Catalog
+title: Action Catalog
 type: actioncatalog
 disable_toc: true
 disable_sidebar: false
 aliases:
-- /workflows/actions_catalog
-- /service_management/service_management/workflows/actions_catalog
-- /service_management/workflows/actions_catalog
+- /workflows/action_catalog
```

Comment:
These aliases usually represent old links that may still exist in the wild and should redirect to this page, so I recommend leaving them as they were.